<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd"> 
<html xmlns="http://www.w3.org/1999/xhtml"> 
 
<head> 
<meta content="text/html; charset=utf-8" http-equiv="Content-Type" /> 
<title>Detecting Population-differentiation Copy Number Variants in Human Population Tree by Sparse Group Selection</title> 
<style type="text/css"> 
.style1 {
	text-align: center;
}
</style> 
</head> 
 
<body style="font-family:Times New Roman, Times, serif"> 
 
<h1 class="style1">Detecting Population-differentiation Copy Number Variants in Human Population Tree by Sparse Group Selection</h1>
<h2 class="style1"><span style="font-weight: 400">Huanan Zhang<sup>1</sup>, David Roe<sup>2,3</sup>  and Rui Kuang<sup>1,3</sup></span></h2> 
<address class="style1">
1. Department of Computer Science and Engineering, University of Minnesota Twin Cities, Minneapolis, MN<BR/>
2. Bioinformatics Research, National Marrow Donor Program, Minneapolis, MN<BR/>
3. Biomedical Informatics and Computational Biology, Minneapolis, MN<BR/>
</address> 
<h3>ABSTRACT</h3> 
<p>Copy-number variants (CNVs) account for a substantial proportion of human genetic variations. Understanding the CNV diversities across populations is a computational challenge because CNV patterns are often present in several related populations and only occur in a subgroup of individuals within each of the population. This paper introduces a tree-guided sparse group selection algorithm (treeSGS) to detect population-differentiation CNV markers of subgroups across populations organized by a phylogenetic tree of human populations. The treeSGS algorithm detects CNV markers of populations associated with nodes from all levels of the tree such that the evolutionary relations among the populations are incorporated for more accurate detection of population-differentiation CNV. We applied treeSGS algorithm to study the 1179 samples from the 11 populations in Hapmap3 CNV genotype data. The treeSGS algorithm accurately identifies CNV markers of each population and the collection of populations organized under the branches of the human population tree, validated by consistency among family trios and SNP characterizations of the CNV regions. Further comparison between the detected CNV markers and other population-differentiation CNVs reported in 1000 genome data and other recent studies also shows that treeSGS can significantly improve the current annotations of population-differentiation CNV markers. Finally, a list of population-differentiation CNVs that are potential Ancestry Informative Markers (AIMs) is reported.</p>
<p><b>Availability: </b>The matlab source code and data is available at [<a href="http://compbio.cs.umn.edu/TreeSGL/code.zip">Source code</a>].</p>
<p><b>Contact: </b>kuang@cs.umn.edu</p>
<p><b>Funding:</b> NSF-IIS 1149697</p>
<BR/>
</body> 
</html> 
